### ISSUE TITLE
No looping in gif as of v1.4.0, regardless of `--no-loop=false`

### ISSUE THREAD

**Issue by PawelLipski** [asciinema/agg#36]
I needed to use ImageMagick (`mogrify -loop 0 <gif>`) to add looping 🤔 

Looping absent both without `--no-loop` and with `--no-loop=false`.

For context, I've used the following `agg` command:

```
theme=000000,ffffff,000000,ff0000,00ff00,ffff00,00ffff,ff00ff,00ffff,d9d9d9,4d4d4d,ff0000,00ff00,ffff00,00ffff,ff00ff,00ffff,ffffff
agg --speed=0.75 --theme="$theme" \
  --cols=122 --rows=35 \
  --font-size=14 --line-height=1.2 \
  "$cast_path" "$gif_path"
```

I can provide more details if needed.

**Comment by LandonTClipp** [comment]
Confirmed that looping behavior works in v1.3.0.

**Comment by LandonTClipp** [comment]
Yes the bug happens in 1.4.0 for me. I meant that the bug does not exist in 1.3.0.

**Comment by Zhengqbbb** [comment]
v1.4.0 gif No looping!

![aaaa](https://user-images.githubusercontent.com/40693636/231340450-cb8f9b54-4bd8-4822-8c7f-8a49b7dccf8d.gif)

**Comment by sickill** [comment]
Hmmkay, this should be easy fix.

**Comment by sickill** [comment]
Fixed in 1.4.1.
